src/services/performanceDbApi.js

```diff
@@ -187,33 +187,23 @@ ORDER BY
     return { datasets }
   },
 
-  /**
-     * Retrieves the resource status from the performance database.
-     *
-     * @param {string} lpa - The Local Planning Authority (LPA) code.
-     * @param {string} datasetId - The ID of the dataset to retrieve the status for.
-     * @returns {object} The resource status object containing the endpoint URL, status, latest log entry date, and days since 200.
-     */
-  getResourceStatus: async (lpa, datasetId) => {
-    const sql =
-    /* sql */
-    `
-    select endpoint_url, status, latest_log_entry_date, days_since_200 from reporting_latest_endpoints
+  resourceStatusQuery (lpa, datasetId) {
+    return /* sql */ `
+    select resource, endpoint_url, status, latest_log_entry_date, days_since_200 
+    from reporting_latest_endpoints
     WHERE REPLACE(organisation, '-eng', '') = '${lpa}'
     AND pipeline = '${datasetId}'`
-
-    const result = await datasette.runQuery(sql)
-
-    return result.formattedData[0]
   },
 
+  datasetIssuesQuery,
```

Comment:
should we define this method in the object? or move other query methods outside the object?

Reply:
this module needs another pass of cleanup, I'll leave it out of this PR